model/src/test/java/jetbrains/jetpad/model/id/BaseIdTest.java

```diff
@@ -46,6 +47,28 @@ public void nameDiscovery () {
     assertEquals("newName1 [a.b]", id2.toString());
   }
 
+  @Test
+  public void idRandomness() {
```

Comment:
It is better to inherit BaseIdTest from BaseTestCase and add @Slow annotation to idRandomness()